stix2elevator/convert_stix.py

```diff
@@ -509,7 +510,7 @@ def finish_markings(instance, env, marking_specifications, temp_marking_id=None)
                                                                         isa_marking,
                                                                         isa_marking_assertions)
                 for m in stix2x_markings:
-                    if m["definition_type"] == "ais":
+                    if "definition_type" in m and m["definition_type"] == "ais":
```

Comment:
You could also do this to be more concise any pythonic:
```suggestion
                    if m.get("definition_type", "") == "ais":
```